File: unical-support-interface/src/app/model/email-dto.ts
import {ClassifyDto} from './classify-dto';
import {AnswerDto} from './answer-dto';

export class EmailDto {
  id: string;
  to: string[];
  subject: string;
  body: string;
  classify: ClassifyDto;
  answer: AnswerDto;

  constructor(data: any) {
    this.id = data.id;
    this.to = data.to;
    this.subject = data.subject;
    this.body = data.body;
    this.classify = data.classify;
    this.answer = data.answer;
  }
}

export { ClassifyDto };

File: unical-support-2/src/main/java/unical_support/unicalsupport2/data/dto/email/EmailDto.java
package unical_support.unicalsupport2.data.dto.email;

import lombok.Data;

import java.util.List;

@Data
public class EmailDto {
    private String id;
    private List<String> to;
    private String subject;
    private String body;
    private ClassifyDto classify;
    private AnswerDto answer;
}
